api/src/tasks/generateNbTxsPerDay.ts

```diff
@@ -15,7 +15,8 @@ export async function generateNbTxsPerDay(currentData: Result[] | undefined) {
   // day the latest data is overwritten
   if (
     currentData &&
-    new Date(currentData[currentData.length - 1].date) === endDate
+    getDate(new Date(currentData[currentData.length - 1].date)) ===
```

Comment:
this could create an issue in case there is a full month to sync (eg: we add one new chart)
Maybe the best is to format the date as a string and compare both strings?